src/backend/src/modules/cssAPI/role/controller.ts

```diff
@@ -65,6 +66,26 @@ export const assignUserRole = errorWrapper(async (req: Request, res: Response) =
   res.json(await assignUserRoles(guid, roleNames));
 });
 
+/**
+ * @method DELETE
+ * @param guid - The user's GUID
+ * @query role - The role to unassign
+ * @route /cssapi/role/assign/:guid
+ * @protected Requires "playground-admin"
+ */
+export const unassignKCUserRole = errorWrapper(async (req: Request, res: Response) => {
+  const { guid } = req.params;
+  const { role } = req.query;
+
+  if (!role || role === '' || typeof role !== 'string')
+    return res.status(404).send("Missing 'role' in request query.");
+
+  if (!guid || guid === '' || typeof role !== 'string')
```

Comment:
```suggestion
  if (!guid)
```
The rest isnt necessary for params, only query.